locales/lt-LT/messages.po

```diff
@@ -0,0 +1,378 @@
+
+msgid "Change since last fingerprint"
+msgstr ""
+
+msgid "Checking the bandwidth..."
+msgstr "Tikrinti pralaidumą..."
+
+msgid "Checking the browser..."
+msgstr "Tikrinti naršklę..."
+
+msgid "Checking the media intensive bandwidth..."
+msgstr "Tikrinamas medijos intensyvumo pralaidumas..."
+
+msgid "Checking the performances..."
+msgstr "Tikrinamas našumas..."
+
+msgid "Checking the screen..."
+msgstr "Tikrinamas ekranas..."
+
+msgid "Checking upload speed..."
+msgstr "Skaičiuojamas įkėlimo greitis..."
+
+msgid "Compatible"
+msgstr "Suderinamas"
+
+msgid "Computing the fingerprint..."
+msgstr "Skaičiuojamas antspaudas..."
+
+msgid "Connect to the diagnostic tool"
+msgstr "Prisijungti prie diagnostikos įrankio"
+
+msgid "Date"
+msgstr "Data"
+
+msgid "Details"
+msgstr "Informacija"
+
+msgid "Diagnostic Tool"
+msgstr "Diagnostikos Įrankis"
+
+msgid "Done!"
+msgstr "Baigta!"
+
+msgid "Dynamic UID"
+msgstr "Dinaminis UID"
+
+msgid "Errors occurred! Please see details"
+msgstr "Įvyko klaidos! Prašome pažiūrėti detales"
+
+msgid "Exit"
+msgstr "Išeiti"
+
+msgid "Failed to get school name"
+msgstr "Nepavyko gauti mokyklos vardo"
+
+msgid "Fingerprint"
+msgstr "Antspaudas"
+
+msgid "Fingerprint error"
+msgstr "Antspaudo klaida"
+
+msgid "Fingerprint errors"
+msgstr "Antspaudo klaidos"
+
+msgid "Fingerprint sources"
+msgstr "Antspaudo šaltiniai"
+
+msgid "Getting school name..."
+msgstr "Gaunamas mokyklos vardas..."
+
+msgid "Good bandwidth"
+msgstr "Geras pralaidumas"
+
+msgid "Good performances"
+msgstr "Geras našumas"
+
+msgid "Good upload speed"
+msgstr "Geras įkėlimo greitis"
+
+msgid "Hide Details"
+msgstr "Slėpti Informaciją"
+
+msgid "Intensive bandwidth"
+msgstr "Intensyvus pralaidumas"
+
+msgid "Interface not available"
+msgstr "Sąsaja nėra galima"
+
+msgid "Launch another readiness check"
+msgstr "Paleisti kitą pasiruošimo patikrinimą"
+
+msgid "Launch readiness check"
+msgstr "Paleisti pasiruošimo patikrinimą"
+
+msgid "List of readiness checks"
+msgstr "Pasiruošimo tikrinimų sąrašas"
+
+msgid "Loading"
+msgstr "Įkeliama"
+
+msgid "Log in"
+msgstr "Prisijungti"
+
+msgid "Login"
+msgstr "Prisijungimas"
+
+msgid "Logout"
+msgstr "Atsijungti"
+
+msgid "Low bandwidth"
+msgstr "Prastas pralaidumas"
+
+msgid "Low upload speed"
+msgstr "Lėtas įkėlimo greitis"
+
+msgid "Max upload speed"
+msgstr "Maksimalus įkėlimo greitis"
+
+msgid "Maximum bandwidth"
+msgstr "Maksimalus pralaidumas"
+
+msgid "Maximum intensive bandwidth"
+msgstr "Maksimalus intensyvus pralaidumas"
+
+msgid "Maximum rendering time"
+msgstr "Maksimalus atvaizdavimo laikas"
+
+msgid "Media intensive bandwidth"
+msgstr "Medijos intensyvus pralaidumas"
+
+msgid "Minimum bandwidth"
+msgstr "Minimalus pralaidumas"
+
+msgid "Minimum intensive bandwidth"
+msgstr "Minimalus intensyvus pralaidumas"
+
+msgid "Minimum rendering time"
+msgstr "Minimalus atvaizdavimo laikas"
+
+msgid "Missing field %s"
+msgstr "Trūkstamas %s laukas "
```

Comment:
additional space on the end of sentence?

Reply:
so in the translation source files

Reply:
In the translation file, there is no space at the end of the sentence.

Reply:
![image](https://user-images.githubusercontent.com/18699340/66580961-e95fe900-eb87-11e9-9ec0-c735eaab6aa8.png)

Reply:
fixed